src/routes/news/components/NewsList.js

```diff
@@ -190,57 +199,61 @@ class NewsList extends React.PureComponent<PropsType> {
                 </HeaderImageWrapper>
               )}
               <Container>
-                <NewsDetailsTitle theme={theme}>
+                <NewsHeadLine theme={theme}>
                   {selectedNewsItem.title}
-                </NewsDetailsTitle>
-                <NewsDetailsContent theme={theme} language={language}>
-                  {content || message}
-                </NewsDetailsContent>
+                </NewsHeadLine>
+                <NewsContent theme={theme} language={language}>
+                  {content}
+                </NewsContent>
               </Container>
+                {isTunews && (
               <Row theme={theme} language={language}>
-                {isTunews ? (
-                  <ExtraInfo theme={theme}>
-                    {`${t('eNewsNo')}: ${eNewsNo}`}
-                    {'  '}
-                    <ExtraInfo onPress={this.openTunewsLink} theme={theme} underlined>tünews INTERNATIONAL</ExtraInfo>
-                    {'  '}
-                    {moment(date).format('MMMM DD, YYYY')}
-                  </ExtraInfo>
-                ) : null}
+                {eNewsNo && typeof eNewsNo === 'string' && <TunewsFooter theme={theme} rightMargin={3}>
+                  {`${t('eNewsNo')}: ${eNewsNo}`}
+                </TunewsFooter>}
+                <TunewsFooter
+                  rightMargin={3}
+                  onPress={this.openTunewsLink}
+                  theme={theme}
+                  underlined>
+                  tünews INTERNATIONAL
+                </TunewsFooter>
+                {isInstanceOfTunews && <MomentContext.Consumer>
+                    {formatter => (
+                      <TunewsFooter theme={theme} rightMargin={3}>
+                        {/* $FlowFixMe this keeps failing no matter the fix is */}
+                        {formatter(selectedNewsItem.date, { format: 'LL', locale: language })}
+                      </TunewsFooter>
+                    )}
+                  </MomentContext.Consumer>}
               </Row>
```

Comment:
Could we move that to a separate component/render method?

Reply:
Still relevant, right?

Reply:
IMO  as I separated the NewsDetails to another component no need to re-separate the footer. What do you think?